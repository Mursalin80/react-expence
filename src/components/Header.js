import React from "react";
import { Link, Router } from "react-router-dom";

const Header = () => (
  <div className="ui menu">
    <header className="header item">
      <h1>Expensify</h1>
      <Router>
        <Link to="/" activeClassName="is-active" exact={true} className="item">
          <div className="ui pramiry primary button">Dashboard</div>
        </Link>
        <Link to="/create" activeClassName="is-active" className="item">
          <div className="ui button">Create Expense</div>
        </Link>
      </Router>
      {/* <NavLink to="/help" activeClassName="is-active">
      Help
    </NavLink> */}
    </header>
  </div>
);

export default Header;
