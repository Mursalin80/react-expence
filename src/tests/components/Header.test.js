import { shallow } from "enzyme";
import React from "react";
import Header from "../../components/Header";
import toJson from "enzyme-to-json";
import "../Enyzme.config";

test("should ", () => {
  let wrapper = shallow(<Header />);
  expect(toJson(wrapper)).toMatchSnapshot();
});
