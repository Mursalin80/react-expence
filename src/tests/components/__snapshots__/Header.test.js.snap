// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should  1`] = `
<div
  className="ui menu"
>
  <header
    className="header item"
  >
    <h1>
      Expensify
    </h1>
    <Router>
      <Link
        activeClassName="is-active"
        className="item"
        exact={true}
        to="/"
      >
        <div
          className="ui pramiry primary button"
        >
          Dashboard
        </div>
      </Link>
      <Link
        activeClassName="is-active"
        className="item"
        to="/create"
      >
        <div
          className="ui button"
        >
          Create Expense
        </div>
      </Link>
    </Router>
  </header>
</div>
`;
